src/components/viewers/MultiFlatmap.vue

```diff
@@ -24,6 +24,35 @@ import EventBus from "../EventBus";
 import store from "../../store";
 import markerZoomLevels from '../markerZoomLevels';
 
+/*
+ * Function to check markers visibility at zoom level.
+ * I have modified it to make sure the marker is displayed
+ * if the uberon is not present in the hardcoded zoom-level list.
+ */
```

Comment:
Might want to rephrase:
```
Check markers' visibility at the given zoom level.
```
The rest of the comment is already captured in this PR (although I would have expected a GitHub issue for this PR).